docs-src/docs/kognic_auth.md

```diff
@@ -32,6 +32,9 @@ There are a few ways to set your credentials in auth.
 API clients such as the KognicIOClient accept this auth parameter.
 Under the hood, they all use the AuthSession class which is implements a requests session with automatic token refresh.
 
+## Generating Credentials
 
+The credentials file, including the Kognic client ID and the Kognic client secret, can be generated in the [Kognic web application](https://app.kognic.com) by clicking on "Api Credentials..." in the user menu, followed by clicking on the "Generate Credentials" button.
 
+![Generate API credentials app screenshot](../static/img/generate-api-credentials.png)
```

Comment:
I wonder if we should have some kind of John Doe user instead of your name Jonatan

Reply:
Yeah, that would be nice. Do we have any standard name an picture that we use? Otherwise I can just edit the page before I take the screen shot.

Reply:
Changed the image in a separate commit to `master`.